### ISSUE TITLE
DeprecationWarning: loaderUtils.parseQuery()

### ISSUE THREAD

**Issue by ajantsch** [tcoopman/image-webpack-loader#82]
Using image-webpack-loader with webpack 2.2 brings up the following warning:

```
DeprecationWarning: loaderUtils.parseQuery() received a non-string value which can be problematic, see https://github.com/webpack/loader-utils/issues/56
parseQuery() will be replaced with getOptions() in the next major version of loader-utils.
    at Object.parseQuery (.../node_modules/image-webpack-loader/node_modules/loader-utils/index.js:78:3)
```

**Comment by followbl** [comment]
a publish to npm of this warning fix would be greatly appreciated :)

**Comment by jonnnnyw** [comment]
@tcoopman Could you please publish this to npm?

**Comment by tcoopman** [comment]
released 3.3.0

**Comment by liiyuli** [comment]
add " process.noDeprecation = true;  " after " process.env.NODE_ENV = JSON.parse(config.dev.env.NODE_ENV); "
in build=> dev-server.js
